posts/Fabio/2018-07-05-hibernate-ogm-5-4-Beta2-released.adoc

```diff
@@ -0,0 +1,96 @@
+= "Hibernate OGM 5.4.0.Beta2 release"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.Beta2] has been released!
+
+We have several news to announce here. Some of them are:
+
+* We support server side indexes definitions for fast queries with Neo4j.
+* We support scripting and java tasks for Infinispan server.
+* WildFly NoSQL subsystem can be used to configure datastore clients.
+* MongoDB dialect supports now Geometry Collection.
+* You can provide a custom Protocol Buffer schema in Infinispan remote.
+* Infinispan remote supports now all unidirectional collections.
+* Infinispan remote sequence generation was improved.
+
+More detail available in the 
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31678[release notes].
+
+== Neo4j server side indexes definitions
+
+JPA 2.1+ `@Index` annotation is now supported for Neo4j dialect.
+This means that Hibernate OGM will create server side indexes for you, according to the entity definition.
+For instance, given the following entity:
+
+====
+[source,java]
+----
+@Entity
+@Table(indexes = {
+	@Index(columnList = "firstname,lastname"), #<1>
+	@Index(columnList = "middlename"), #<2>
+	@Index(columnList = "nickname", unique = true) #<3>
+})
+private static class Person {
+
+	@Id
+	private String id;
+	private String firstname;
+	private String middlename;
+	private String lastname;
+	private String nickname;
+}
+----
+====
+
+Hibernate OGM will create three indexes:
+
+1. an index, allowing duplicates, on field pair (firstname, lastname)
+2. an index, allowing duplicates, on field middlename
+3. an index, not allowing duplicates, on field nickname
+
+[NOTE]
+The use of JPA annotation `@Index` always imply that the indexes will be stored on server side, within the datastore.
+If you need to store indexes on client side or move that somewhere else, you can instead use Hibernate Search.
```

Comment:
I would remove this note about Hibernate Search, I feel it could be confusing for people not overly familiar with it

Reply:
I was uncertain about this sentence. I'll be going to remove it.